Framework/MDAlgorithms/test/ConvertToMDTest.h

```diff
@@ -89,7 +89,7 @@ class ConvertToMDTest : public CxxTest::TestSuite {
     TS_ASSERT_THROWS_NOTHING(pAlg->initialize())
     TS_ASSERT(pAlg->isInitialized())
 
-    TSM_ASSERT_EQUALS("algorithm should have 25 properties", 25,
+    TSM_ASSERT_EQUALS("algorithm should have 26 properties", 26,
```

Comment:
default vs indexing has no sense in my opinion, or I don't know the metric to use for comparison, the tree structure may be different a little, but I have no good idea, how to describe this "little". Single threaded vs multithreaded may have a sense.

Reply:
Is it different in all cases? If most or identical we could still compare?

In case all are different, can we converted `MDEventWorkspace` -> `MDHistoWorkspace` and then do a comparison with a small epsilon? This will it least show that the big picture is consistent.

Reply:
I can't say in "all cases", I can construct a case with the identical tree structure, but it is quite complicated to construct the synthetic case for old algorithm because of monolith design the only option is to generate kind of real data for real or mode (if there is one) instrument. Are where some scripts for generating such kind of data? For histograms it may work, then it's better to add tests on python level using CompareWS algorithm.

Reply:
Ok, so if it is quite common the the files we have I suggest we consider the second option I suggested, i.e., comparing after conversion to histogram?

You can use `CompareWorkspaces` in C++ by the way, but if you prefer doing it in Python that is just as good.